runtime/services/catalog/artifacts/yaml/objects.go

```diff
@@ -145,9 +143,6 @@ func fromSourceArtifact(source *Source, path string) (*drivers.CatalogEntry, err
 	if source.Region != "" {
 		props["region"] = source.Region
 	}
-	if source.CsvDelimiter != "" {
-		props["csv.delimiter"] = source.CsvDelimiter
-	}
```

Comment:
Can we keep reading `csv.delimiter` from the source artifact, and writing it to `duckdb.delim` here instead – for backwards compatibility